.travis.yml

```diff
@@ -19,6 +19,11 @@ script:
   - docker-compose up --build --force-recreate -d
   - sleep 20
   - wget --no-check-certificate https://eid.local
+  - docker-compose exec api python manage.py creatersakey
+  - docker-compose exec api python manage.py shell -c "$(cat test/e2e/spec/oidc_provider/create_django_user.py)"
```

Comment:
Replace this with a Django command, cf a [similar task in desec-stack](https://github.com/desec-io/desec-stack/blob/master/api/desecapi/management/commands/privacy-chores.py)